counsel.el

```diff
@@ -5136,25 +5136,26 @@ to variable to `savehist-additional-variables'.")
     counsel-compile-get-make-invocation)
   "Additional compile invocations to feed into `counsel-compile'.
 
-This can either be a list of compile invocations strings or
+This can either be a list of compile invocation strings or
 functions that will provide such a list.  You should customise
 this if you want to provide specific non-standard build types to
-`counsel-compile'.  The default helpers are set up to handle common
-build environments.")
+`counsel-compile'.  The default helpers are set up to handle
+common build environments.")
 
 (defcustom counsel-compile-make-args "-k"
   "Additional arguments for make.
-You may for example want to add -jN for the number of cores you have."
+You may, for example, want to add \"-jN\" for the number of cores
+N in your system."
   :type 'string)
 
-(defcustom counsel-compile-make-pattern "\\(?:GNUM\\|[Mm]\\)akefile"
-  "Pattern for matching against makefiles."
-  :type 'regex)
+(defcustom counsel-compile-make-pattern "\\`\\(?:GNUM\\|[Mm]\\)akefile\\'"
```

Comment:
huh, I missed that rx usage had been dropped. I guess no love for rx then...

Reply:
Personally, I always use `rx`. I find its advantages far outweigh its disadvantages. Note, however, that #1941 made suboptimal and even incorrect usage of `rx`.